staging/src/k8s.io/api/testdata/HEAD/core.v1.Service.json

```diff
@@ -71,7 +71,8 @@
       "clientIP": {
         "timeoutSeconds": -1487653240
       }
-    }
+    },
+    "serviceIPFamily": "Ơ歿:狞夌碕ʂɭîcP$I"
```

Comment:
encoding error?

Reply:
no.. these things are auto generated. they are used in roundtripper.

Reply:
btw out of abundance if caution i tried to check if the letter ^^^ means something inappropriate and couldn't. Google translate couldn't translate them. So please if you find it so, let me know i can talk with folks who wrote the generator to fix it.